src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -467,24 +467,25 @@
         }
 
         .datagrid-column-handle {
-          @include clr-no-styles-button();
-          display: block;
           position: absolute;
           width: calc(0.5rem + #{$clr-rem-1px});
-          right: -0.5rem;
+          right: -0.25rem;
```

Comment:
Did removing the button change the offset values?

Reply:
This was wrong value initially. The separator underneath the resize handle should be exactly aligned at the middle.

**Before:**
![screenshot 2018-12-21 14 54 12](https://user-images.githubusercontent.com/3958008/50366734-699d5b00-0530-11e9-8c77-c389e004a8df.png)

**After:**
![screenshot 2018-12-21 14 56 13](https://user-images.githubusercontent.com/3958008/50366766-9487af00-0530-11e9-92ec-4fc9bee6b3fe.png)